projects/kit/tokens/tag-status.ts

```diff
@@ -1,10 +0,0 @@
-import {InjectionToken} from '@angular/core';
-import {TuiStatus} from '@taiga-ui/kit/types';
-
-// TODO: 3.0 use TUI_TAG_OPTIONS instead of it
-export const TUI_TAG_STATUS = new InjectionToken<TuiStatus>(
```

Comment:
oops, I have made a mistake.

I've mistakenly moved it to `TUI_INPUT_TAG_OPTIONS` (not TUI_TAG_OPTIONS).

Reply:
I'll fix it soon)

Reply:
fixed ✅